go/api/generic_commands.go

```diff
@@ -13,7 +13,7 @@ type GenericBaseCommands interface {
 	// Del removes the specified keys from the database. A key is ignored if it does not exist.
 	//
 	// Note:
-	//When in cluster mode, the command may route to multiple nodes when `keys` map to different hash slots.
+	//  When in cluster mode, the command may route to multiple nodes when `keys` map to different hash slots.
```

Comment:
Please apply new note here as well

Reply:
It's covered in this PR: https://github.com/valkey-io/valkey-glide/pull/2738